pages/03.Howtos/docs.de.md

```diff
@@ -0,0 +1,10 @@
+---
+title: How-to guides
+page-toc:
+    active: false
+taxonomy:
+    category:
+        - how-to
```

Comment:
Shouldn't it be the same category as the english variant?
I saw some other category name "howtos". Should be consistent.

Reply:
```suggestion
        - docs
```